### PR TITLE
SHOT-4293: Fix PySide6 patch for QPalette.Background and PySide import ordering

### PR DESCRIPTION
**Pull request by staceyoue**
* QPalette.Background is now removed in Qt6. Use QPalette.Window instead (as per Qt doc)
* Revert PySide import ordering in QtImporter to try to import PySide2 first and PySide6 last

### CODE REVIEW

**Review by julien-lang — APPROVED** [review]
LGTM!

**Review by carlos-villavicencio-adsk — APPROVED** [review]
LGTM

File: tests/util_tests/test_pyside6_patcher.py
Comment (staceyoue) [reply tests/util_tests/test_pyside6_patcher.py:43]: @julien-lang is hound not in sync with the Azure Pipelines black style? The CI code validation passes, but hound still complains here. Do we still need hound if we have it in the CI already?

**Comment by staceyoue** [comment]
@julien-lang this is ready for a final review. I have added unit tests to cover the qt importer and patch functions and has increased overall coverage, but codecov is still complaining. Any advice on what to do with that? We cannot merge without codecov approval.

**Review by julien-lang — APPROVED** [review]

**Pull request merged**
